tool/teleport/common/teleport.go

```diff
@@ -96,6 +96,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 		fmt.Sprintf("Comma-separated list of roles to start with [%s]", strings.Join(defaults.StartRoles, ","))).
 		Short('r').
 		StringVar(&ccf.Roles)
+	start.Flag("join-method", "Method used for new nodes to join the cluster").StringVar(&ccf.JoinMethod)
```

Comment:
I like to use and `EnumVar` for things like this, see https://github.com/gravitational/teleport/blob/master/tool/tbot/main.go#L86

```suggestion
	start.Flag("join-method", "Method to use to join the cluster, can be `"token\", \"ec2\", or \"iam\"").EnumVar(&ccf.JoinMethod, "token", "ec2", "iam")
```

Reply:
That's much better, thanks